Add migration notes to wix forms v1 components (#3358)
wix-incubator/wix-code-docs: wix-formbuilder/$w/WixForms/guides/Migrate from WixForms.md

# Migrate from WixForms to WixFormsV2
As of April 2025, [WixForms](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/introduction) is deprecated and any new forms created will only work with [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction). You can access the functionality of WixForms using [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction).

> Old forms will only work with the old APIs, WixForms, WixFormSubmitEvent, WixFormSubmittedErrorEvent, and WixFormSubmittedEvent. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, there's a tab for each. There is no easy way to distinguish in the editor, except that their menus are different.

Reviewer: ```suggestion
Old forms will only work with the old APIs: `WixForms`, `WixFormSubmitEvent`, `WixFormSubmittedErrorEvent`, and `WixFormSubmittedEvent`. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, where there's a tab for each. 
```
Please check that everything that should be in code font is in code font throughout (e.g. `WixForms`, `WixFormsV2`)

Also, why not link to the old APIs since you are mentioning them? 

(Same comments apply for the other places this paragraph appears.)

Below is a list of the deprecated WixForms components and functions and their alternatives in WixFormsV2.

### [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/collapsed)

> Instead of `collapsed`, use [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/collapsed) from WixFormsV2.

Reviewer: ```suggestion
Instead of `collapsed` from `WixForms`, use [`collapsed`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/collapsed) from `WixFormsV2`.
```
I think being explicit about the fact that the deprecated one is from `WixForms` is especially important when the equivalent property in V2 has the same name -- hence the suggestion to add "from `WixForms`". If already making this change for those with identical names, I think we may as well word this way throughout.

### [`hidden`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/hidden)
Instead of `collapsed`, use [`hidden`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/hidden) from WixFormsV2.

### [`isVisible`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/is-visible)
Instead of `collapsed`, use [`isVisible`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/is-visible) from WixFormsV2.

### [`onWixFormSubmit()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/on-wix-form-submit)
Instead of adding `onWixFormSubmit()` to run when a site visitor starts to submit a Wix Form yet before the form is actually submitted and sent to the server, use [`onSubmit()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/on-submit) from WixFormsV2.

### [`onWixFormSubmitted()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/on-wix-form-submitted)

> Instead of adding `onWixFormSubmitted()` to run when a site visitor starts to submit a Wix Form yet before the form is actually submitted and sent to the server, use [`onSubmitSuccess()`](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/on-submit-success) from WixFormsV2.

Reviewer: This description is incorrect - it seems you accidentally replicated the previous one.
